hibernate-core/src/test/java/org/hibernate/orm/test/mapping/formula/FormulaTests.java

```diff
@@ -94,7 +96,8 @@ void tearDown(SessionFactoryScope scope) {
 
 		private Double rate;
 
-		@Formula(value = "credit * rate")
+		@Formula(value = "credit * rate",
+				overrides = @DialectOverride(dialect = H2Dialect.class, value ="rate * credit") )
 		private Double interest;
```

Comment:
Here's an example of how you might use it, but it's much more general than this.